package.json

```diff
@@ -409,6 +409,13 @@
                   "type": "string"
                 },
                 "default": []
+              },
+              "env": {
```

Comment:
[question]
would it make sense to keep it in alignment with how java env are defined? 
use Map[String, String] ?

Reply:
what do you mean? `jvmOpts` parameter is the same.

Reply:
ahh sorry it was mentioned in issue this fixes:
https://code.visualstudio.com/docs/java/java-debugging
https://code.visualstudio.com/docs/java/java-debugging/cwd-env.mp4

Reply:
yeah lol I was far from being precise, keep it the same as in java extension that's what I meant xD

Reply:
We could do it, I will take a look tomorrow.

Reply:
Added!